Remove unclear paragraph about font-size values

UlrichThomasGabor opened the pull request:
…

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation

This text position was unclear previously and is now more precise and more in line with the rest of the document.

<!-- ❓ Why are you making these changes and how do they help readers? -->

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

UlrichThomasGabor committed:
web: Made more clear to what percentage values in `font-size` property refer to

estelle commented on files/en-us/web/css/font-size/index.md:
```diff
@@ -49,7 +49,7 @@ font-size: unset;
 
 - As one of the absolute-size, relative-size or `math` keywords
-- As a `<length>` or a `<percentage>`, relative to the element's font size.
+- As a `<length>` or a `<percentage>`, relative to the parent element's font size.
```
How about we switch the order so it is clear that iit is only the percentage that is relative to the parent element's size, and the length is not specified (as what a relative length value is relative to depends on the unit). How about this:

```suggestion
- As a `<percentage>` relative to the parent element's font size or a `<length>`.
```

estelle replied:
alternatively, i think it might be better to get ride of line 49-52 and replace it with:

```suggestion
The `font-size` property is specified as a `<length>`, `<percentage>`, or an `<absolute-size>`, `<relative-size>` or the `math` keyword.
```

UlrichThomasGabor replied:
Why not remove the whole paragraph including the list (or just the list)?
<img width="646" alt="Bildschirmfoto 2024-07-11 um 09 58 27" src="https://github.com/mdn/content/assets/18024904/ae138256-786e-40a2-9c5e-8fedfd4c16af">

Because it is basically a summarized enumeration of the directly following `Values` section.

While our suggestions improve the situation by making the "summary" more accurate – on a closer look I do not see how this paragraph provides much value.
- It is obviously hard to succinctly AND correctly shorten the description of the different values. (Your suggestion 1 seems correct to me, but leaves open what a `length` is. Your suggestion 2 overlaps with the first bullet point, therefore I do not think this really improves the text.)
- Like the `Values` section, it is basically a list. Same structure, same values, shortened description. Since they are so close together, I do not really see the benefit of this descriptions if it is hard to make it right.

If some kind of transition to the next section should stay, I think I would go with a more coarse summary:
```markdown
The `font-size` property can be specified as an absolute or relative value in various
ways. See the following `Values` section for details.
```

estelle replied:
> Your suggestion 1 seems correct to me, but leaves open what a `length` is. 

Length is defined along with all the values in the section, so maybe we can go with option 1, but move it after the "values" header, instead of before it.

>Your suggestion 2 overlaps with the first bullet point, therefore I do not think this really improves the text.)

The second suggestion is _instead_ of the paragraph and two bullet points.

> If some kind of transition to the next section should stay, I think I would go with a more coarse summary:
> 
> ```
> The `font-size` property can be specified as an absolute or relative value in various
> ways. See the following `Values` section for details.
> ```

This doesn't add information, so another alternative is to just remove the sentence and the two bullet points. the explanation of percent occurs two more times in the doc.

estelle replied:
> alternatively, i think it might be better to get ride of line 49-52 and replace it with:

This suggestion is to replace the paragraph and two bullets, not simply a replacement for just the paragraph

UlrichThomasGabor replied:
Since we both seem fine with just removing the paragraph, I have removed it. Seems the more robust solution to me.

UlrichThomasGabor committed:
Removed paragraph

because it is misleading, does not add real value and the `Values` section follows directly

estelle reviewed (approved):
Thank you. 
Also, congratulations on your first merged MDN content PR. 
Welcome to the team!
